source/blueprints/banking.rst

```diff
@@ -26,45 +26,41 @@ Here are some examples of scripted dialogue interactions for conversational flow
-
+   Pick a convenient textual or graphical representation. Try to design as many user flows as you can. Always capture the entire dialogue from start to finish. Think of scenarios that differ from the examples above, such as: trying to transfer money you don't have, reporting a fraudulent charge on your credit card, asking for your routing number, and so on. 
 
 3. Domain-Intent-Entity Hierarchy
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Here is the NLP model hierarchy for our food ordering application.
-
-.. image:: /images/food_ordering_hierarchy.png
-    :width: 700px
-    :align: center
-
-The single domain, ``ordering``, encompasses all of the functionality required to order food for delivery from nearby restaurants. The ``ordering`` domain supports the following intents:
+The banking blueprint is organized into five domains: ``Greeting``, ``Accounts & Credit Cards``, ``FAQ``, and ``Unsupported``. The main domain, ``accounts_creditcards``, encompasses all of the functionality required to perform tasks that require personal user banking information. The ``accounts_creditcards`` domain supports the following intents:
 
    - ``greet`` — User wants to start a conversation
-   - ``build_order`` — User wants to make selections for food delivery
-   - ``place_order`` — User wants to confirm and place an order
-   - ``start_over`` — User wants to abandon current selections and restart the ordering process
+   - ``transfer_balances`` — User wants to transfer money from one account to another
+   - ``check_balances`` — User wants the current balance for a particular account
+   - ``pay_creditcard`` — User wants to pay off a credit card bill
+   - ``check_due_date`` — User wants the date the bill is due 
+   - ``setup_autopay`` — User wants to activate automatic payments
    - ``exit`` — User wants to end the current conversation
    - ``help`` — User is confused or needs instructions on how to proceed
-   - ``unsupported`` — User is talking about something other than food ordering
+   - ``unsupported`` — User is talking about something other than banking tasks
+
+The full list of intents for all domains is illustrated below.
+
+.. image:: /images/hierarchy_home_assistant.png
 
-For this app, only the ``build_order`` intent requires entity recognition. This intent supports the following entity types:
+There are two types of entities in MindMeld: :ref:`System Entities <system-entities>` and :doc:`Custom Entities <../userguide/entity_recognizer>`. System entities are pre-defined in MindMeld. Examples include ``sys_amount-of-money``, ``sys_time``, and ``sys_interval``. Custom entities are defined by the developers of each application. Within each entity folder, the file ``gazetteer.txt`` contains the full list of values for each custom entity.
 
```

Comment:
Update to: There are two types of entities in MindMeld: :ref:`System Entities <system-entities>` and :doc:`Custom Entities <../userguide/entity_recognizer>`. System entities are pre-defined and automatically detected by MindMeld. Examples include ``sys_amount-of-money``, ``sys_time``, and ``sys_interval``. Custom entities are defined by the developers of each application and detected by the models trained on the labeled data and entity gazetteer files.